manifests/o/OpenShot/OpenShot/2.5.1/OpenShot.OpenShot.installer.yaml

```diff
@@ -0,0 +1,32 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: OpenShot.OpenShot
+PackageVersion: 2.5.1
+MinimumOSVersion: 10.0.0.0
+FileExtensions:
+- .osp
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: inno
+  Scope: machine
+  InstallerUrl: https://github.com/OpenShot/openshot-qt/releases/download/v2.5.1/OpenShot-v2.5.1-x86_64.exe
+  InstallerSha256: CB9B7EA7B72D8147B85D0F881DE819E4F81092F6CAC605BE2E61753DA6DC9026
+  UpgradeBehavior: install
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: inno
+  Scope: machine
+  InstallerUrl: https://github.com/OpenShot/openshot-qt/releases/download/v2.5.1/OpenShot-v2.5.1-x86.exe
+  InstallerSha256: 9FE1CB6BD0B220DA72E7A6B79BBC57E00AD295C04CC896DFBE58A43EA9401837
+  InstallerSwitches:
+    Custom: en-US
```

Comment:
```suggestion
```
Typo? Was going to suggest /ALLUSERS and /CURRENTUSER but seems like the installer is only All Users

Reply:
A typo, of course. I entered it in the YamlCreate script one line too early. My bad.